### ISSUE TITLE
Accessibility audit - Severe warnings

### ISSUE THREAD

**Issue by Akin909** [FAC10/week1-Facalacalaca#58]
On running an audit it flags up the use of the same IDs on multiple elements which if being used by a screen reader would repeat this title by which it identifies the element confusing the user as to what part of the page they were navigating through

The text area at the bottom of the contact form is not labelled which means the contents of the text area would not be read by a screen reader
